api/app/settings/common.py

```diff
@@ -291,14 +291,12 @@
 if GOOGLE_ANALYTICS_KEY:
     MIDDLEWARE.append("app_analytics.middleware.GoogleAnalyticsMiddleware")
 
-if INFLUXDB_TOKEN:
-    MIDDLEWARE.append("app_analytics.middleware.InfluxDBMiddleware")
 
 if USE_POSTGRES_FOR_ANALYTICS:
-    if INFLUXDB_BUCKET:
-        raise RuntimeError("Cannot use both InfluxDB and Postgres for analytics")
-
     MIDDLEWARE.append("app_analytics.middleware.APIUsageMiddleware")
+elif INFLUXDB_TOKEN:
+    MIDDLEWARE.append("app_analytics.middleware.InfluxDBMiddleware")
+
```

Comment:
As per discussions in slack, I think we should add a comment to try and explain the logic here. I think it would be good to logically group all of the influx settings here too.

Reply:
Yeah, Done